src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -347,6 +347,46 @@ def get_login_credentials(self, resource=None, client_id=None, subscription_id=N
                 str(account[_TENANT_ID]))
 
     def get_raw_token(self, resource=None, scopes=None, subscription=None, tenant=None):
```

Comment:
`get_raw_token` is only designed for getting the raw token. It cannot be used with Python SDK.

To invoke Python SDK, it is required to call `get_login_credentials` to get the credential.